README.rst

```diff
@@ -101,8 +139,66 @@ Run tests
 
     make tests
 
+
+Running in production
+=====================
+
+Recommended settings
+--------------------
+
+::
+
+    cliquet.batch_max_requests = 25
```

Comment:
Do we need to specify `cliquet.batch_max_requests` or is the default 25, per https://github.com/mozilla-services/readinglist/pull/185/files#diff-050fd003f97cbf6523c1103f07a9295bR16 ?